src/dropdown/dropdown.component.ts

```diff
@@ -70,12 +69,13 @@ import { ElementService } from "./../utils/utils.module";
 			[tabindex]="disabled ? -1 : 0">
 			<div
 				(click)="clearSelected()"
+				(keydown.enter)="clearSelected()"
 				*ngIf="type === 'multi' && getSelectedCount() > 0"
-				class="bx--list-box__selection--multi"
+				class="bx--list-box__selection bx--tag--filter bx--list-box__selection--multi"
```

Comment:
Same note here about `tag--filter` ... also do we want `selection` and `selection--multi` at the same time? Or is that something we should `ngClass`?

Reply:
I'm not sure if we need both since this clear selected button only pops up for multi-select. I was following the structure in react's implementation and they had both, however it seems to work without the `selection` class